src/connectors/numeric-selector/connectNumericSelector.js

```diff
@@ -160,6 +160,46 @@ export default function connectNumericSelector(renderFn, unmountFn) {
         return state.removeNumericRefinement(attributeName);
       },
 
+      getWidgetState(fullState, { state }) {
```

Comment:
According to the change in `getWidgetSearchParameters`:

- `fullState` should be renamed `uiState`
- `state` should be renamed `searchParam`